docs/versioned_docs/version-v1.0.0-beta.3/how_to/how-to-solidity-verifier.mdx

```diff
@@ -140,14 +132,14 @@ Barretenberg attaches the public inputs to the proof, which in this case it's no
 ```bash
 PROOF_HEX=$(cat ./target/proof | od -An -v -t x1 | tr -d $' \n' | sed 's/^.\{8\}//')
 
-NUM_PUBLIC_INPUTS=2
+NUM_PUBLIC_INPUTS=1  # Replace this with the number of public inputs in your circuit
 HEX_PUBLIC_INPUTS=${PROOF_HEX:0:$((32 * $NUM_PUBLIC_INPUTS * 2))}
-SPLIT_HEX_PUBLIC_INPUTS=$(sed -e 's/.\{64\}/0x&,/g' <<<$HEX_PUBLIC_INPUTS)
+SPLIT_HEX_PUBLIC_INPUTS=$(sed -e 's/.\{64\}/"0x&",/g' <<<$HEX_PUBLIC_INPUTS)
 
 PROOF_WITHOUT_PUBLIC_INPUTS="${PROOF_HEX:$((NUM_PUBLIC_INPUTS * 32 * 2))}"
 
 echo 0x$PROOF_WITHOUT_PUBLIC_INPUTS
-echo [$SPLIT_HEX_PUBLIC_INPUTS]
+echo "[${SPLIT_HEX_PUBLIC_INPUTS}]"
 ```
```

Comment:
It would be really good to have this use snippets to pull from the `examples` directory

Reply:
True.
But this one is no longer required with latest bb, and is already removed from latest docs.